vue3_ts/src/views/veevalidate-yup/ManuallyErrors.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="max-w-450 mx-auto my-30">
+    <h1 class="mb-30 text-center">Manually errors example</h1>
+
+    <form class="mb-30" @submit="onSubmit">
+      <div class="flex flex-col mb-10">
+        <input
+          v-model="email"
+          type="email"
+          name="email"
+          class="px-10 py-5 border border-grey-400 rounded"
+        >
+        <span class="inline-block mt-5 text-red-500">{{ emailError }}</span>
+      </div>
+
+      <div class="flex flex-col">
+        <input
+          v-model="password"
+          type="password"
+          name="password"
+          class="px-10 py-5 border border-grey-400 rounded"
+        >
+        <span class="inline-block mt-5 text-red-500">{{ passwordError }}</span>
+      </div>
+
+      <button class="mt-30 py-5 px-10 rounded-sm text-white bg-grey-800">
+        Example submit
+      </button>
+    </form>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue'
+import { veevalidateYupService } from './veevalidate-yup.service'
+
+import { useForm, useField } from 'vee-validate'
+import { loginSchema } from './veevalidate-yup.schemas'
```

Comment:
please create a folder called schemas on the same level and principles like for example types